.github/workflows/release.yml

```diff
@@ -0,0 +1,53 @@
+name: Release
+
+# Runs manually
+on:
+  workflow_dispatch:
+      
+jobs:
+  release:
+    name: Release on Sonatype OSS
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Apache Maven Central
+        uses: actions/setup-java@v3
+        with:
+          java-version: 11
+          distribution: 'temurin'
+          cache: 'maven'
+          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }} # Value of the GPG private key to import
+          gpg-passphrase: GPG_PASSPHRASE # env variable for GPG private key passphrase
+
+      - name: Publish to Apache Maven Central
+        run: |
+          maven_config="$(cat .mvn/maven.config)"
+          snapshot_version="${maven_config%% *}"
+          suffix="-SNAPSHOT"
+          release_version=${snapshot_version%"$suffix"}
+          sed -i "s/${snapshot_version}/${release_version}/" .mvn/maven.config
+          mvn --settings .mvn/settings.xml deploy -Pdeploy,release -Dgpg.passphrase=${{ secrets.GPG_PASSPHRASE }}
+        env:
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+
+      # Create github release with the specified tag, name and body
+      - name: create gitHub release
+        run: |
+          maven_config="$(cat .mvn/maven.config)"
+          version="${maven_config%% *}"
+          prefix="-Drevision="
+          release_version=${version#"$prefix"}
+          noDotVersion="${release_version//.}"
+          gh release create ${version} --notes "# Download
+          https://repo1.maven.org/maven2/com/devonfw/tools/ide/devonfw-ide-scripts/${version}/devonfw-ide-scripts-${version}.tar.gz
+          # Changes
+          https://github.com/devonfw/ide/blob/master/CHANGELOG.asciidoc#${noDotVersion}"
+          git tag Release/"${version}"
+          git push origin test Release/"${version}"
```

Comment:
why `test` ? Was that just temporary for tests in your fork?
https://github.com/devonfw/ide/blob/d250ffa520daa74bf0068737c5c0f6191c93fe95/scripts/src/main/resources/scripts/command/release#L135
https://github.com/devonfw/ide/blob/d250ffa520daa74bf0068737c5c0f6191c93fe95/scripts/src/main/resources/scripts/command/release#L136